scripts/EFI_icprocess.R

```diff
@@ -0,0 +1,6 @@
+
+# EFI_icprocess -----------------------------------------------------------
+
+#required inputs
+
+
```

Comment:
This file does nothing -- does it need to be added now or should it wait until it contains code?

Reply:
Yes that is a mistake I thought I removed that file. Should definitely wait to add it until it contains code.